python/GafferSceneUI/RenderPassEditor.py

```diff
@@ -174,7 +174,7 @@ def registerOption( cls, groupKey, optionName, section = "Main", columnName = No
 		toolTip = "<h3>{}</h3>".format( optionLabel or columnName )
 		optionDescription = Gaffer.Metadata.value( "option:" + optionName, "description" )
 		if optionDescription :
-			toolTip += "\n\n" + optionDescription
+			toolTip += optionDescription.replace( "\n\n", "<br/><br/>" )
```

Comment:
Perhaps `DocumentationAlgo.markdownToHTML()`? And a `\todo` that PathListingWidget's PathModel should be doing that for us?

Reply:
Swapped that over and added a `\todo` in 4b836d5c4f6eedf26b4fc1a54f380a770d75cd15.